build.sbt

```diff
@@ -270,9 +268,6 @@ val previousVersions = Seq("1.6.0")
 
 val noMima = mimaPreviousArtifacts := Set.empty
 val mimaSettings: Seq[Setting[_]] = {
-  import ProblemFilters.exclude
-  import sbt.librarymanagement.SemanticSelector
-  import sbt.librarymanagement.VersionNumber
```

Comment:
Hmm, this was magically removed by either IntelliJ's formatter or `scalafmtSbt`. Or maybe I invoked an `Organize imports` while the focus was on `build.sbt`. 

Nothing's broken though...